app/components/publications/publication/translations/translation-request-modal.hbs

```diff
@@ -1,74 +1,106 @@
 <Auk::Modal>
   <Auk::Modal::Header
+    @resizeable={{false}}
     @title={{t
       "request-translations-for-publication"
       publicationNumber=@publicationFlow.identification.idName
     }}
-    @onClose={{@onCancel}}
+    @onClose={{perform this.cancel}}
   />
   <Auk::Modal::Body>
-    <Utils::LoadableContent
-      @isLoading={{this.saveRequest.isRunning}}
-      @reserveSpace={{true}}
-    >
-      <div class="auk-form-group-layout auk-form-group-layout--standard">
-        {{! Due date }}
-        {{! TODO: Remove inline style }}
-        {{! template-lint-disable no-inline-styles }}
-        <div class="auk-form-group" style="max-width: 30rem;">
-          <Auk::Label for="translationRequestDueDate">
-            {{t "publication-flow-translation-due-date"}}
-          </Auk::Label>
-          <Auk::Datepicker
-            id="translationRequestDueDate"
-            @date={{this.translationDueDate}}
-            @onChange={{this.setTranslationDueDate}}
-          />
-        </div>
-        {{! template-lint-enable no-inline-styles }}
```

Comment:
`{{! template-lint-enable no-inline-styles }}`
Deze mag wel blijven staan. Andere, minder bewust ingevoegde inline styles worden dan nog steeds opgemerkt. Zelfde voor `translation-requested-panel.hbs`.

Reply:
Ik ga in de nabije toekomst hier even kort een item voor opnemen rond:
- al dan niet verschil tussen `form-group` en `input-group` (laatste zal misschien verdwijnen aangezien die niets doet).
- mogelijks toe voegen van width modifier classes (in de aard van `.auk-form-group--w-medium`) om zo inline styling te vermijden.

Indien die laatste doorgevoerd wordt doorloop ik sowieso nog eens alle gevallen van `class="auk-form-group"` waar er inline styling aan te pas komt.